src/db/schema/satelite.ts

```diff
@@ -0,0 +1,10 @@
+import mongoose from 'mongoose'
+import Utilities from '../../utils/utils'
+
+const headings = Utilities.getUsableHeadingsFromFile()
+const finalObj = Object.fromEntries(headings.map((h) => [h, String]))
```

Comment:
Better name

Reply:
What does this line do

Reply:
Its schema creation from 1 line of txt file, eg. { prop1: String, prop2: String etc }

Reply:
Call it with a name related to schema